### PR TITLE
Small docs additions

### PR DESCRIPTION
**Pull request by jabbacakes**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Small additions to the docs following our discussion, including doc updates related to [PR #3004](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/3004) in the main Netcode for GameObjects code repo.

### CODE REVIEW

File: docs/basics/networkobject.md [hunk 1]
```diff
@@ -88,6 +88,33 @@ GetComponent<NetworkObject>().SpawnAsPlayerObject(clientId);
 
 If the player already had a prefab instance assigned, then the client owns the NetworkObject of that prefab instance unless there's additional server-side specific user code that removes or changes the ownership.
 
+### Defining defaults for PlayerObjects
```
Comment (jabbacakes) [inline docs/basics/networkobject.md:91]: @NoelStephensUnity

File: docs/basics/networkvariable.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ Additionally, any type that has a managed type is itself a managed type - so a s
 
 Finally, while managed `INetworkSerializable` types are serialized in-place (and thus don't incur allocations for simple value updates), C# arrays and managed types serialized through custom serialization are **not** serialized in-place, and will incur an allocation on every update.
 
+### Using collections with `NetworkVariable`s
```
Comment (jabbacakes) [inline docs/basics/networkvariable.md:73]: @NoelStephensUnity

File: docs/components/networkanimator.md [hunk 3]
```diff
@@ -134,11 +131,17 @@ public void ApplyMotion(Vector3 playerVelocity)
...
-For triggers you always want to use `NetworkAnimator`. One example might be that you use a trigger, called it " `IsJumping`, " to start a blended transition between the player's walking/running animation and the jumping animation:
+For triggers, you always want to use `NetworkAnimator`. One example might be that you use a trigger, called " `IsJumping`, " to start a blended transition between the player's walking/running animation and the jumping animation:
 
 ```csharp
 public void SetPlayerJumping(bool isJumping)
 {
     m_NetworkAnimator.SetTrigger("IsJumping");
 }
 ```
+
```
Comment (jabbacakes) [inline docs/components/networkanimator.md:142]: @NoelStephensUnity

**Comment by jabbacakes** [comment]
@NoelStephensUnity Doc updates from our meeting on Tuesday. I did a bit of polishing while I was at it, so I've highlighted the three new key bits with comments.

File: docs/basics/networkvariable.md [hunk 4]
```diff
@@ -254,7 +260,7 @@ There are two options for reading a `NetworkVariable.Value`:
 
 There are two options for writing a `NetworkVariable.Value`:
 
-- **Server(default)**: the server is the only one that can write the value.
+- **Server (default)**: the server is the only one that can write the value.
```
Comment (NoelStephensUnity) [inline docs/basics/networkvariable.md:263]: We might make a note here that when using a distributed authority network topology the write permissions will automatically be updated to owner write permissions (even if they have set server write permissions). This automatic conversion is just to help covert a project started with a client-server network topology (i.e. conversion) and/or for projects that want to support both client-server and distributed authority network topologies.

**Pull request merged**